Police-Data-Accessibility-Project/data-sources-app-v2
# Create endpoint for following/unfollowing searches, as well as gettin…

### Fixes

* https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/476

### Description

* Create endpoints for following/unfollowing searches, as well as getting all searches for a specific user

#### LIBTYFI:
* Replace prior database client create/update/delete Postgres logic with (more concise) SQLAlchemy logic.

### Testing

* Run tests in `tests` directory. Confirm all behave as expected.
* Check API to confirm presence of expected documentation.

### Performance

* Conversion to SQLAlchemy imposes minor performance cost.

### Docs

* API documentation updated.

### Breaking Changes

* No breaking changes.
